buyer-api/src/blockchain/dataOrder.js

```diff
@@ -39,5 +39,5 @@ export async function fetchDataOrder(dxId) {
       closedAt: toDate(closedAt),
     };
   }
-  throw new Error(`Invalid DataExchange-Id ${dxId}`);
```

Comment:
I know understand the comment @Meketh did regarding throwing errors here. I think we should avoid them since web3 is already doing the same check (verify if `orderId` is a number).